deis-dev/manifests/deis-workflow-manager-rc.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: deis-workflow-manager
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  replicas: 1
+  selector:
+    app: deis-workflow-manager
+  template:
+    metadata:
+      labels:
+        app: deis-workflow-manager
+    spec:
+      serviceAccount: deis-workflow-manager
+      containers:
+      - name: deis-workflow-manager
+        image: quay.io/deisci/workflow-manager:v2-beta
+        imagePullPolicy: Always
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: WORKFLOW_MANAGER_PORT
+          value: "8080"
+        - name: WORKFLOW_MANAGER_VERSIONSAPIURL
+          value: "https://versions.deis.com"
+        - name: WORKFLOW_MANAGER_STAGINGAPIURL
+          value: "https://versions-staging.deis.com"
+        - name: WORKFLOW_MANAGER_DOCTORAPIURL
```

Comment:
this and the previous 2 env vars look like they're missing a few underscores. is that purposeful?